### ISSUE TITLE
Support weave to enable multicast between pods and into pods from outside the cluster

### ISSUE THREAD

**Issue by stodge** [k3s-io/k3s#1287]
**Is your feature request related to a problem? Please describe.**
I am unable to broadcast multicast packets between pods in my k3s cluster.

**Describe the solution you'd like**
I am using k3s in a development environment to simplify developing with kubernetes. I am porting a number of existing monolithic Java services to micro services.

One of these services subscribes to a multicast group and decodes packets. Another service (simulator) broadcasts multicast packets, which the former service can optionally subscribe to.

Basically, I need the ability to broadcast multicast between pods. I looked through the k3s repository and tried reading about Weave, but I quickly realised it's way over my head.

**Describe alternatives you've considered**
I haven't considered any alternatives, as I don't think there are any.

Thanks!

**Comment by thomasferrandiz** [comment]
This should be possible without changes in k3s itself by using multus and a dedicated interface on each pod for the multicast traffic.
